utils/network.go

```diff
@@ -10,7 +10,10 @@ import (
 	"github.com/spf13/pflag"
 )
 
-var ErrUnknownNetwork = errors.New("unknown network (known: mainnet, goerli, goerli2, integration)")
+var (
+	ErrUnknownNetwork = errors.New("unknown network (known: mainnet, goerli, goerli2, integration)")
+	requiredSpace     = map[string]float64{"mainnet": 38.0, "goerli": 34.0, "goerli2": 6.5}
```

Comment:
Who is going to update this as the chain grows?

Can it not be calculated, even very roughly, from the current chain height and the storage used per block? (We'd need this to execute after we know the current chain height though.)

Reply:
I brought that into discussion on how can we know the size of the chain in advance. Tho, I didn't see an approach for that rather than hard-code it and update it by time. 
Your suggestion seems great, we can know the current chain height by fetching the latest block, but what about `storage used per block`, is there a way to get it ? And if there's, I assume we need to take kind of an average, do you suggest an approach to calculate that average ?

Reply:
The chain size is very arbitrary as `mainnet` and `goerli` are roughly the same size but the difference is the number of blocks is around 650,000.

We can calculate the average size of the block and multiply it with the remaining blocks to be synced.

 Given this is a rough calculation, we may stop someone from syncing when there is enough disk space due to an inaccurate estimate.